sw/host/opentitanlib/src/io/spi.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+
+/// Represents the SPI transfer mode.
+pub enum TransferMode {
+    Mode0,
```

Comment:
Are these just temporary or are they intended to actually mean something?

Reply:
Sadly, these are the de-facto names of the SPI transfer modes.  They refer to the various bit combos of CPOL and CPHA (clock polarity and phase) and control when the data is driven and sampled.

Reply:
Maybe some comments are in order?

Reply:
Done.